app/DailyMenu.php

```diff
@@ -7,15 +7,67 @@
 
 class DailyMenu extends Model
 {
-    use softDeletes;
+    use SoftDeletes;
 
+    protected $timestamp = true;
+    protected $dates = ['deleted_at'];
     /**
-     * DailyMenu has many Food
+     * The table associated with the model.
      *
-     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     * @var string
      */
-    public function foods()
+    protected $table = 'daily_menus';
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = ['quantity', 'created_at', 'updated_at'];
+
+    /**
+     * Get the food for the menu item.
+     *
+     * @return object
+     */
+    public function food()
+    {
+        return $this->belongsTo('App\Food', 'food_id', 'id');
+    }
+
+    /**
+     * Return all menu item in database with paginate by 10 item per page
+     *
+     * @return array Menu object
+     */
+    public function getAll()
+    {
+        return self::select('date')
```

Comment:
You should move this code to controller and use pagination.
-example: $this->dailyMenu->select()....->pagination(10)